src/averageAges.js

```diff
@@ -37,7 +40,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter((person) => {
+    const hasChild = people.some((child) => child.mother === person.name);
+
+    return (person.sex === 'f' && withChildren)
+      ? person.sex === 'f' && hasChild
+      : person.sex === 'f';
+  });
```

Comment:
Same as in `calculateMenAverageAge` function

Reply:
👌 fixed